### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by GeorgeTsendra**

### CODE REVIEW

File: homework-2/js.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ 1. Составить предложение из нижестоящих переменных:
+ "Сколько нужно программистов чтобы сделать проект ? 1, 25, команда"
+ */
+
+let numbers = [25, 1];
+let project = 'проект';
+let team = `команда`;
+let howMuch = 'Сколько';
+let sentence = 'нужно программистов чтобы сделать проект ?';
+let task1 = howMuch + sentence.replace("проект", project)+ " "+numbers[0]+","+" "+ numbers[1]+","+" " +team  ;
```
Comment (OlegLustenko) [inline homework-2/js.js]: I guess it's better to use 'es6 string' it would make such code much readable

File: homework-2/js.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  'он обнаружил',
+  'не',
+  'Так, когда Будда достиг Просветления',
+  '...',
+  'что больше',
+  'ощущает себя мишенью',
+  'не'
+];
+
+let homeSentence = array[4] + ", " + array[2]+ ", " + array[6] + " " + array[8] + " " + array[7] + ". " + "\n"+array[0]
```
Comment (OlegLustenko) [inline homework-2/js.js]: same here about es6 string concatenation

File: homework-2/js.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ 3. Добавьте свойста 4 новых свойства в объект используя
+    квадратные скобки и точку.
+ */
+
+let myObj = {};
+myObj.first = 120;
+myObj.second = ["Hello"];
+myObj.second[1] = "World";
+myObj.therd = "easy";
+myObj.forth = 12990;
```
Comment (OlegLustenko) [inline homework-2/js.js]: you have added some value to array it's good, but also you have to use **[]** and add property to object

File: homework-2/js.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+// console.log(myObj);
+
+/*
+ 5. Преобразуйте строку x,
+    в максимально удобно читаемый для программиста вид
+ */
+
+let frameworks = [4.7, 'Angular', '6Angular', 'React/Redux'];
+let x ='google\ released ' +'new version\ ' + frameworks[1] + Math.floor(frameworks[0]) +
```
Comment (OlegLustenko) [inline homework-2/js.js]: I guess it could be improved, have you tried to use es6 string interpolation ?

Comment (GeorgeTsendra) [reply homework-2/js.js]: ok, I just think that I cannot change the code.

**Comment by OlegLustenko** [comment]
another PR don't need. I would be enough to add 1 more commit to that branch it would give us history of every changes

File: homework2/js.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+myObj[fff] = 2222;
+
+// console.log(myObj);
+/*
+ 5. Преобразуйте строку x,
+    в максимально удобно читаемый для программиста вид
+ */
+
+let frameworks = [4.7, 'Angular', '6Angular', 'React/Redux'];
+let x =`google released new version ${frameworks[1]} ${Math.floor(frameworks[0])} But real speed is ${frameworks[frameworks.length - 1]}`;
```
Comment (OlegLustenko) [inline homework2/js.js]: what do you think can we improve such constructions?
```js
${frameworks[frameworks.length - 1]}
```

Comment (GeorgeTsendra) [reply homework2/js.js]: hm.... may be will be beter to use ${frameworks[3]}

Comment (OlegLustenko) [reply homework2/js.js]: not really, I believe if move it to another variable which would tell us, what does it really mean

**Pull request merged**